components/Cards/ResourceCard/ResourceCard.js

```diff
@@ -63,20 +65,19 @@ function ResourceCard({
 
   // Sync IDs with stylesheet
   // eslint-disable-next-line react/prop-types
-  const VotingBlock = ({ id }) => {
-    const onUpvote = () => handleVote('upvotes', setUpVotes);
-    const downUpvote = () => handleVote('downvotes', setDownVotes);
+  const VotingBlock = ({ blockID }) => {
+    const onVote = voteDirection => handleVote(voteDirection, id, setUpVotes, setDownVotes);
```

Comment:
I still actually prefer the separate methods so that we're not defining anonymous methods that don't compile down (see line 80 and 105).

Feel free to keep `onVote`, but I still think defining:

```js
const onUpvote = onVote('upvote');
const onDownvote = onVote('downvote');
```

is best practice regarding the mantra of always naming your methods.

Reply:
okay no problem. I'll make separate functions for each.